journal/modernized/2-journal.adoc

```diff
@@ -7727,6 +7762,11 @@ and by whom it is believed, these pages will be read with interest and instructi
 [.signed-section-context-close]
 New York, Ninth month 7th, 1812.
 
+//not sure if this above should be embedded^^
```

Comment:
Ok, yeah, confusing.  I thought it was one thing, but then I read it again, and now I think we probably should embed it on it's own.  Not sure what _type_ to give it though... Let's leave it blank for now, just `[.embedded-content-document]`